lib/reporters/tap.js

```diff
@@ -7,7 +7,7 @@ function getSourceFromStack(stack, index) {
 		.split('\n')
 		.slice(index, index + 1)
 		.join('')
-		.replace(/^\s+at /, '');
```

Comment:
I meant, previously we were removing `at` from the "standard" stack trace:

```js
Error: damn it
  at Test.fn (file.js:4:1)
```

But now the replacing regex does not have it.